cep47/src/cep47_storage.rs

```diff
@@ -117,42 +79,47 @@ impl CEP47Storage for CasperCEP47Storage {
         data::update_total_supply(new_total_supply);
     }
 
+    fn transfer_many(&mut self, sender: &Key, recipient: &Key, token_ids: &Vec<TokenId>) {
+        // Prepare dictionaries.
+        let owners_dict = Owners::instance();
+        let balances_dict = Balances::instance();
+
+        // Update ownerships.
+        for token_id in token_ids {
+            owners_dict.set(token_id, *recipient);
+        }
```

Comment:
Does it automatically update that `sender` does not own the token anymore?

Reply:
No.
It will change the ownership of that token to another account.